CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ BREAKING CHANGES
 * modules/mesh-task: The retry_join variable was updated to take a list of
   members rather than a single member.
   [[GH-59](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/59)]
+* modules/mesh-task: Default the `accessor_id` and `token` stored on the
+  service token. This change allows `consul-ecs` to fix a number of small bugs
+  related to ACL token creation and deletion.
+  [[GH-65](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/65)]
```

Comment:
I think I want to reposition this as part of the overall ACL controller bug fix.

Maybe include this as part of a single bullet point for ACL controller bug fixes, combining this with L38?

```
* modules/acl-controller and modules/mesh-task: Fix bugs in which ACL tokens
  were not created or deleted in certain cases.
  [[GH-63](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/63)]
  [[GH-65](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/65)]
```

Thoughts?